CHANGELOG.md

```diff
@@ -29,14 +29,24 @@ _No documentation available about unreleased changes as of yet._
 - Moved the `WordPress.CodeAnalysis.EmptyStatement` sniff from the `WordPress-Extra` to the `WordPress-Core` ruleset.
 - Moved the `Squiz.PHP.CommentedOutCode` sniff from the `WordPress-Docs` to the `WordPress-Extra` ruleset and lowered the threshold for determining whether or not a comment is commented out code from 45% to 40%.
 - The `WordPress.NamingConventions.PrefixAllGlobals` sniff now has improved support for recognizing whether or not (non-prefixed) globals are declared in the context of unit tests.
+- The `is_foreach_as()` method has been moved from the `GlobalVariablesOverrideSniff` class to the WordPress `Sniff` base class.
 - The `Sniff::is_token_in_test_method()` utility method now has improved support for recognizing test methods in anonymous classes.
 - Minor efficiency improvement to the `Sniff::is_safe_casted()` method.
 - CI: Minor tweaks to the Travis script.
 - CI: Improved Composer scripts for use by WPCS developers.
 - Readme: Improved the documentation about the project history and the badge display.
 
 ### Fixed
-- `DeprecatedClasses`: The errorcodes for this sniff were unstable as they were based on the code being analysed instead of on fixed values.
+- The `WordPress.WhiteSpace.PrecisionAlignment` sniff will no longer throw false positives for docblocks for javascript functions within inline HTML.
+- `WordPress.WP.DeprecatedClasses`: The errorcodes for this sniff were unstable as they were based on the code being analysed instead of on fixed values.
+- Various bugfixes for the `WordPress.WP.GlobalVariablesOverride` sniff:
+    - Previously, the sniff only checked variables in the global namespace when a `global` statement would be encountered. As of now, all variable assignments in the global namespace will be checked.
+    - Nested functions/closures/classes which don't import the global variable will now be skipped over when encountered within another function, preventing false positives.
+    - Parameters in function declarations will no longer throw false positives.
+    - The error message for assignments to a subkey of the `$GLOBALS` superglobal has been improved.
+    - Various efficiency improvements.
+- The `Sniff::is_in_isset_or_empty()` method presumed the WordPress coding style regarding code layout, which could lead to incorrect results (mostly underreporting).
+    The affected a.o. the `WordPress.Security.ValidatedSanitizedInput` sniff.
```

Comment:
"a.o." ?

Reply:
amongst others

Reply:
I vaguely recall there is one other sniff using that function under the hood, I just didn't feel like tracing the back through the code to see which one :stuck_out_tongue_winking_eye:

Reply:
a.o. is not a common abbreviation in English. Please expand:

`This affected, amongst others, the `WordPress.Security.ValidatedSanitizedInput` sniff.

Reply:
> Please expand

I already did in this morning's update ;-) (Though only just noticed that I still had to change the `The` to `This`)